packages/doenetml-worker-rust/lib-rust-macros/src/component_node.rs

```diff
@@ -22,8 +36,7 @@ pub fn component_node_derive(input: TokenStream) -> TokenStream {
                     n => n.to_case(Case::Camel),
                 };
 
-                quote! {
-                    impl ComponentNode for #name {
+                let mut component_node_impl_body = quote! {
                         fn get_idx(&self) -> ComponentIdx {
```

Comment:
These lines got indented twice, and the formatter doesn't work inside `quote!` to fix it.